tests/scripts/expect/posix-ccathreshold.exp

```diff
@@ -39,6 +39,8 @@ expect_line "Done"
 send "\x04"
 expect eof
 
+sleep 0.2
```

Comment:
can we try to avoid such delays?

Reply:
I have removed the delays. I added the destructor of `RadioSpinel` to free the occupied resource to make ot-rcp to free the socket resource to avoid adding delays here.